benchmarks/.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
```

Comment:
I think this file can be removed

Reply:
No, we need it as `.build` is otherwise included. I don't actually mind this additional .gitignore

Reply:
I think we should add that to the root `.gitingore` instead of having nested ones. WDYT?

Reply:
I'm not a fan of that. If we ever going to remove the benchmarks folder we would need to remember to edit the parent `.gitignore`. I quite like that git can have nested gitignores and I don't really see a downside to it. Why do you think this should live in the parent `.gitignore`?

Reply:
I don't have a strong opinions. I personally find nested `.gitignores` a bit weird.